README.md

```diff
@@ -1,8 +1,142 @@
 # trading_calendars
 
 ![CI](https://github.com/quantopian/trading_calendars/workflows/CI/badge.svg)
```

Comment:
Let's make this link to the workflows instead of to the image?
```suggestion
[![CI](https://github.com/quantopian/trading_calendars/workflows/CI/badge.svg)](https://github.com/quantopian/trading_calendars/actions?query=workflow%3ACI)
```

(Not for this PR, but looks like no CI workflow is running on master, or I would have filtered by branch=master.)